packages/date-adapters/src/index.ts

```diff
@@ -1,18 +1,5 @@
-/**
- * To add a new adapter, then, add the adapter's date object to `DateFrameworkTypeMap` interface
- *
- * declare module "./types" {
- *   interface DateFrameworkTypeMap {
- *     luxon: DateTime;
- *   }
- * }
- */
-// biome-ignore lint/complexity/noBannedTypes: type augmented by configured adapters
-export type DateFrameworkTypeMap = {};
-
-export * from "./date-fns";
-export * from "./dayjs";
-export * from "./luxon";
-export * from "./moment";
-
+export * from "./date-fns-adapter";
```

Comment:
Are we keeping this just for our own benefit? Or do we expect it to be used by users?

Reply:
removed and refactored site/tests